server/src/create-svg.es6

```diff
@@ -48,20 +29,31 @@ const addForestSpecificInfo = (permit, frag) => {
 
   frag.querySelector('#permit-year-vertical').textContent = permit.christmasTreesForest.startDate.getFullYear();
 
-  frag.querySelector(
-    '#permit-harvest-expiration'
-  ).textContent = `THIS PERMIT EXPIRES AT MIDNIGHT OF THE HARVEST DATE FILLED IN BELOW OR ${moment(
-    permit.christmasTreesForest.endDate,
-    util.datetimeFormat
-  )
-    .format('MMM D, YYYY h:mm A')
-    .toUpperCase()}`;
   frag.querySelector('#permit-expiration').textContent = moment(
     permit.christmasTreesForest.endDate,
     util.datetimeFormat
   )
     .format('MMM D, YYYY h:mm A')
     .toUpperCase();
+  if (permit.christmasTreesForest.treeHeight > 0) {
```

Comment:
if code climate complains about this set all as default then change to N/A if missing